src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
```

Comment:
You can format your function something like this if the function does not fit in one line
```suggestion
    ? people.filter((p) => {
      return p.sex === 'f' && people.find((p1) => p1.mother === p.name)
    })
```